bridge/client/app/_components/ktb-sli-breakdown/ktb-sli-breakdown.component.ts

```diff
@@ -95,30 +95,21 @@ export class KtbSliBreakdownComponent implements OnInit {
       this.columnNames = ['details', 'name', 'value', 'weight', 'passTargets', 'warningTargets', 'result', 'score'];
     }
     return indicatorResults.map((indicatorResult) => {
-      let accSum = 0;
-      let accCount = 0;
-      for (const comparedIndicatorResult of this.comparedIndicatorResults) {
-        const result = comparedIndicatorResult.find((res) => res.value.metric === indicatorResult.value.metric);
-        if (result) {
-          accSum += result.value.value;
-          accCount++;
-        }
-      }
-      const comparedValue = accSum / accCount;
+      const comparedValue = indicatorResult.value.comparedValue ?? this.calculateComparedValue(indicatorResult);
       const compared: Partial<SliResult> = {};
-      if (comparedValue) {
+      if (!isNaN(comparedValue)) {
         compared.comparedValue = AppUtils.formatNumber(comparedValue);
         compared.calculatedChanges = {
-          absolute: AppUtils.formatNumber(comparedValue - indicatorResult.value.value),
-          relative: AppUtils.formatNumber((comparedValue / (indicatorResult.value.value || 1)) * 100 - 100),
+          absolute: AppUtils.formatNumber(indicatorResult.value.value - comparedValue),
+          relative: AppUtils.formatNumber((indicatorResult.value.value / (comparedValue || 1)) * 100 - 100),
         };
       }
 
       return {
         name: indicatorResult.displayName || indicatorResult.value.metric,
         value: indicatorResult.value.message || AppUtils.formatNumber(indicatorResult.value.value),
         result: indicatorResult.status,
-        score: totalscore === 0 ? 0 : AppUtils.round((indicatorResult.score / totalscore) * this.score, 2),
+        score: totalscore === 0 ? 0 : (indicatorResult.score / totalscore) * this.score,
```

Comment:
With this change, there may be different values for the score (SLI-breakdown and overlay of an evaluation). It should be the same for both of them. Also, the value is affected.
https://github.com/keptn/keptn/blob/246753a3e7e63342482378e7e1b5eef206dbbcc0/bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts#L589-L615
https://github.com/keptn/keptn/blob/246753a3e7e63342482378e7e1b5eef206dbbcc0/bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.html#L69

Reply:
Now we use the same truncateNumber for all places were the score is displayed.
![image](https://user-images.githubusercontent.com/6098219/157422579-101dfd5e-5c77-49ad-ae36-53734c353453.png)

Reply:
As discussed, we will truncate the score for the badges to 0 decimals.

![image](https://user-images.githubusercontent.com/6098219/157454984-4274ecd5-8793-4307-b919-4704e1a0896d.png)